style/extensions.txt

```diff
@@ -810,14 +810,15 @@ extensions must define two additional tokens.
     specification, and is incremented when changes are made.
     Note that the revision of an extension defined in the Vulkan header
     files and the revision supported by the Vulkan implementation (the
-    pname:specVersion field of the slink:VkExtensionProperties structure
+    pname:specVersion field of the sname:VkExtensionProperties structure
     corresponding to the extension and returned by one of the
-    <<extended-functionality-extensions,extension queries>>) may differ.
+    link:html/vkspec.html#extendingvulkan-extensions[extension queries])
+    may differ.
     The revision value indicates a patch version of the extension
     specification, and differences in this version number maintain full
     compatibility, as defined in the
-    link:html/vkspec.html#fundamentals-versionnum[API Version Numbers and
-    Semantics] section of the <<vulkan-spec,Vulkan API Specification>>.
+    link:html/vkspec.html#_compatibility_guarantees_informative[Compatibility Guarantees]
+    section of the <<vulkan-spec,Vulkan API Specification>>.
```

Comment:
PS: technically I think the link does not say the same, so perhaps worth a spec issue.